test/unit/store/item-store.js

```diff
@@ -75,4 +75,25 @@ describe('ItemStore', () => {
         });
     });
 
+    it('should delete an item', () => {
+        assert.strictEqual(items.length, 3);
+        return addAction('DELETE_ITEM', {
+            itemAddress: items[0].address
+        }).then(() => {
+            items = ItemStore.getItems();
+            assert.strictEqual(items.length, 2);
+        });
+    });
+
+    it('should fail to delete an item', () =>{
+        assert.strictEqual(items.length, 2);
+        return addAction('DELETE_ITEM', {
+            itemAddress: 'This is not an address'
+        }).then(() => {
+            throw new Error('Expected to catch error since there was not a valid address given to delete item');
```

Comment:
I would change this to `assert(false)`, instead of throwing the error to make it more explicit, but I don't think that's a blocker if this test is working.